UnityProject/Assets/Scripts/Objects/Research/Artifacts/Artifact.cs

```diff
@@ -70,7 +70,7 @@ public class Artifact : ImnterfaceMultitoolGUI, IServerSpawn, IServerDespawn, IC
 
 		private Coroutine animationCoroutine = null;
 
-		[SyncVar, PlayModeOnly] public ArtifactData artifactData = new ArtifactData();
+		[SyncVar/*, PlayModeOnly*/] public ArtifactData artifactData = new ArtifactData();
```

Comment:
Why has this been commented out on this and the artifact ID?

Reply:
temp changes as it was causing compile errors for me, it will be reverted before marked as ready